blue_hardware_drivers/include/blue_hardware_drivers/BLDCDriver.h

```diff
@@ -24,14 +24,19 @@ constexpr int MAX_TEMP_SHUTOFF = 70;
 class BLDCDriver {
   public:
     void init(std::string port, std::vector<comm_id_t> board_ids);
-    void update(std::unordered_map<comm_id_t, float>& commands, blue_msgs::MotorState& motor_states);
+    void update(
+        std::unordered_map<comm_id_t, float>& pos_commands,
+        std::unordered_map<comm_id_t, float>& current_commads,
+        blue_msgs::MotorState& motor_states);
     void engageControl();
     void disengageControl();
+    void setControlMode(size_t id, comm_ctrl_mode_t control_mode);
```

Comment:
should this actually be `comm_id_t` and not `size_t`? (my bad for misunderstanding earlier)